libraries/node-core-library/src/FileSystem.ts

```diff
@@ -1068,8 +1068,8 @@ export class FileSystem {
       ...options
     };
 
-    await FileSystem._wrapExceptionAsync(async () => {
-      fsx.copySync(options.sourcePath, options.destinationPath, {
+    await FileSystem._wrapExceptionAsync(() => {
+      return fsx.copy(options.sourcePath, options.destinationPath, {
```

Comment:
```suggestion
    await FileSystem._wrapExceptionAsync(async () => {
      await fsx.copy(options.sourcePath, options.destinationPath, {
```

That will give a cleaner stack trace.